docs/workflows/stories-for-multiple-components.md

```diff
@@ -38,7 +38,8 @@ The simplest change we can make to the above is to reuse the stories of the `Lis
   paths={[
     'react/list-story-unchecked.js.mdx',
     'react/list-story-unchecked.ts.mdx',
-    'vue/list-story-unchecked.js.mdx'
+    'vue/list-story-unchecked.js.mdx',
+    'angular/list-story-unchecked.ts.mdx'
```

Comment:
```suggestion
    'angular/list-story-unchecked.ts.mdx',
```

Reply:
Fixed with the latest commit. Thank you for checking it :)